src/components/Contact/contactlist.tsx

```diff
@@ -0,0 +1,41 @@
+import ContactItem from '@/components/Contact/contactitem'
+
+type contactData = {
+  id: string
+  first_name: string
+  last_name: string
+  pets: string
+  email: string
+  phone: string
+  street_address: string
+  region: string
+  notes: string
+}
+
+const ContactList = ({ contacts }) => {
+  const contactsItems = []
+
+  for (let i = 0; i < contacts.length; i++) {
+    contactsItems.push(
+      <ContactItem
+        key={contacts[i].id}
+        userid={contacts[i].id}
+        image={contacts.image}
+        name={`${contacts[i].first_name} ${contacts[i].last_name}`}
+      />
+    )
+  }
```

Comment:
A cleaner way to do this would be using the map() function:
contactsItems = contacts.map((contact) ->(
  <ContactItem
    key={contact.id}
    userid={contact.id}
    .......
   />
)

https://reactjs.org/docs/lists-and-keys.html. It achieves the same thing but is the cleaner way of doing it

Reply:
Chantelle has just fixed this